js/hooks/useNfc.ts

```diff
@@ -31,15 +47,16 @@ export const useNfc = (cancel: AbortController): NfcResult => {
               status: "success",
               data: dehexSerial(event.serialNumber),
             });
-            cancel.abort();
+
+            abortController.abort();
```

Comment:
Why call this on success?

Reply:
Because on success we want the reader to stop listening for NFC taps. It's a weird edge case, but because of the old bug in Chrome that caused us to not use the abort controller in Glides, if you do a badge tap sign-in and then later on in the flow tap a badge on your phone again with the browser still open, it will make the little vibration indicating that the NFC reader is reading. With this call to `abortController.abort()`, it will stop reading once we get a successful scan and won't start reading again until you start a new sign-in flow and a new copy of the `useNfc` hook is mounted.

Reply:
Would you mind putting that exact explanation in as a comment? Got it.

Reply:
I edited it slightly to better fit the context but here: 23ebe1a